plugins/header_rewrite/header_rewrite.cc

```diff
@@ -523,6 +548,8 @@ TSRemapDoRemap(void *ih, TSHttpTxn rh, TSRemapRequestInfo *rri)
     if (rule->eval(res)) {
       OperModifiers rt = rule->exec(res);
 
+      ink_assert((rt & OPER_NO_REENABLE) == 0);
```

Comment:
Generally speaking, ink_assert() should not be used in plugin code.  But, since header_rewrite already include core ATS headers, go with the flow I guess.